app/models/metrics_updates.rb

```diff
@@ -0,0 +1,9 @@
+class MetricsUpdates
+
+  def self.last_update(time)
```

Comment:
This method is a setter, so maybe call it `last_update=` or `log_update` or something else that communicates what the method does.

Relatedly, `last_update` should probably be the getter that you put in `course.rb`.